readthedocs/templates/projects/project_import.html

```diff
@@ -70,7 +70,18 @@
       </span>
 
       <h1>{% trans "Import a Repository" %}</h1>
-
+      {% if has_connected_accounts %}
+        {% block import-repo %}
```

Comment:
The block name could be a little more descriptive, like "private-repo-tip" or similar.